### PR TITLE
4730 Interface Tagging . Don't check the Elements that cannot contain layers inside

### LINKED ISSUE #4730: Handling large track files
I'm working with some large track files.

Two large tracks are in the attached zip file.  The first one takes a few seconds to load, but the second one takes ages.  I _Think_ it's because it's doing a screen redraw after loading each new point.

I can think of two way around this:
* cache up all the positions for a track in the ImportReplay class, and add them at once at the end.
* load the points into a new track, but only add the track to the `Layers` entity once we've finished loading the file
* it _may_ be an option to introduce the ability to pause the `Layers` object from sending out modified/extended/reformatted events.  Then, un-pause it, and send out an extended event manually.

[Archive.zip](https://github.com/debrief/debrief/files/4078077/Archive.zip)

### PR DESCRIPTION
**Pull request by saulhidalgoaular**
Fix #4730

### CODE REVIEW

File: org.mwc.cmap.legacy/src/MWC/GUI/ICantContainLayers.java [hunk 1]
```diff
@@ -0,0 +1,6 @@
+package MWC.GUI;
+
```
Comment (IanMayo) [inline org.mwc.cmap.legacy/src/MWC/GUI/ICantContainLayers.java]: @saulhidalgoaular - could you add a comment explaining the role of this interface?

Comment (saulhidalgoaular) [reply org.mwc.cmap.legacy/src/MWC/GUI/ICantContainLayers.java]: Sure. 1 Minute

File: org.mwc.cmap.legacy/src/MWC/GUI/Layers.java [hunk 2]
```diff
@@ -488,7 +488,12 @@ private static Layer checkLayer(final Layer layer, final String name)
         {
           return l;
         }
-        else if ( !(l instanceof ICantContainLayers) )
```
Comment (IanMayo) [inline org.mwc.cmap.legacy/src/MWC/GUI/Layers.java]: @saulhidalgoaular  - I've slightly restructured, so I can document why we have that path through the logic

It won't be a problem, will it?

**Comment by IanMayo** [comment]
Nice work Saul, the tracks now load _in a flash_.

File: org.mwc.cmap.legacy/src/MWC/GUI/Layers.java (hunk 2 above)
Comment (saulhidalgoaular) [reply org.mwc.cmap.legacy/src/MWC/GUI/Layers.java]: Your change is perfect!!!!

**Pull request merged**